utils/download.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+# Copyright 2021  Jiayu Du
+#                 Seasalt AI, Inc (Author: Guoguo Chen)
+#                 Tsinghua University (Author: Shuzhou Chai)
+#                 Mobvoi Inc (Author: Binbin Zhang)
+
+set -e
```

Comment:
Consider choosing one from ```set -e``` or  ```exit``` in the shell.
If you don't catch the non-zero exit code, I don't think explicit ```1``` is necessary although a lot of people do that.